tools/test-backend

```diff
@@ -123,7 +123,7 @@ def block_internet():
                         "https://zulip.readthedocs.io/en/latest/testing.html#internet-access-inside-test-suits")
 
     httplib2.Http.request = internet_guard
-    requests.request = internet_guard
+    requests.request = internet_guard  # type: ignore # FIXME file bug in mypy repo!
```

Comment:
@timabbott See https://github.com/zulip/zulip/pull/6017#discussion_r131818592 for previous discussion -- because the signature of `internet_guard` is already super general, I think this is actually a bug in mypy we should file.

(Not a blocker for merging, though -- now that we've had some discussion, I can just tweakt the comment as appropriate on merge.)